pkg/fatfs/Makefile.dep

```diff
@@ -10,7 +10,7 @@ include $(RIOTBASE)/boards/$(BOARD)/Makefile.features
 
 #if periph_rtc is available use it. Otherwise use static timestamps
 ifneq (, $(filter periph_rtc, $(FEATURES_PROVIDED)))
-   export CFLAGS+= -DFATFS_FFCONF_OPT_FS_NORTC=0
+  CFLAGS += -DFATFS_FFCONF_OPT_FS_NORTC=0
 else
-	export CFLAGS+= -DFATFS_FFCONF_OPT_FS_NORTC=1
+   CFLAGS += -DFATFS_FFCONF_OPT_FS_NORTC=1
```

Comment:
indent off

Reply:
Well that is the correct one (non-command indentations in makefiles should be 2 spaces)

Reply:
Github shows line 15 with three spaces.

Reply:
Ah ^^" well I think I go back to bed then.